src/V3Width.cpp

```diff
@@ -3326,15 +3346,15 @@ class WidthVisitor final : public AstNVisitor {
             } else if (VN_IS(dtypep, BasicDType) && VN_CAST(dtypep, BasicDType)->isRanged()) {
                 VL_DO_DANGLING(patternBasic(nodep, dtypep, defaultp), nodep);
             } else {
-                nodep->v3warn(
-                    E_UNSUPPORTED,
-                    "Unsupported: Assignment pattern applies against non struct/union data type: "
-                        << dtypep->prettyDTypeNameQ());
+                nodep->v3warn(E_UNSUPPORTED, "Unsupported: Assignment pattern applies "
+                                             "against non struct/union data type: "
+                                                 << dtypep->prettyDTypeNameQ());
```

Comment:
Is this now an error?

Reply:
Did somehow my addition changed the behavior in this case?

Reply:
ok I think